docs/build/assets/arrow.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These are files that are generated in the build process and should not be included.

Reply:
Ohh. I'll delete them